### ISSUE TITLE
Microsoft Voice Access engine implementation for Dragonfly

### ISSUE THREAD

**Issue by drmfinlay** [dictation-toolbox/dragonfly#388]
Microsoft announced late last year that Windows Speech Recognition (WSR) is now deprecated in favour of Voice Access, a new feature of Windows 11.  WSR may be removed in a future build or release of Windows.  It would be nice if Dragonfly could use Voice Access through an API.  I'll do some testing to see how feasible this is.

**Comment by drmfinlay** [comment]
Either way, I think it would be good to have an engine back-end that uses the newer .NET speech API.  [This](https://learn.microsoft.com/en-us/dotnet/api/system.speech.recognition?view=dotnet-plat-ext-8.0) one.

**Comment by drmfinlay** [comment]
Microsoft have said that WSR will be removed from Windows 11 in an update in September, 2024.  I presume that means that the older API we use -- SAPI 5, deprecated for many years now -- will also finally be removed.  I suppose we'll find out in September.

**Comment by kendonB** [comment]
I tried Voice Access and found the recognition very bad. My opinion is that Dragonfly should deprecate WSR support in favour of kaldi.

**Comment by drmfinlay** [comment]
I agree, I don't think there's much point bothering with this.  There is clearly very little interest.

Kaldi is a better overall choice, for English.  Dragon and WSR are undoubtedly better for other languages.  There is also CMU Sphinx, which is acceptable for small grammars.
